controllers/replica_set_controller.go

```diff
@@ -407,6 +435,23 @@ func (r *ReplicaSetReconciler) createOrUpdateStatefulSet(mdb mdbv1.MongoDBCommun
 	return nil
 }
 
+func (r *ReplicaSetReconciler) createOrUpdateStatefulSetForArbiters(mdb mdbv1.MongoDBCommunity) error {
```

Comment:
Wondering if we can re-use `createOrUpdateStatefulSet` instead of adding a different function for arbiters?